### PR TITLE
Adding VstsCreateMergePR to Azure Functions

### PR DESCRIPTION
**Pull request by Shyam-Gupta**
This new Azure function will automatically create code flow PRs across VSUnitTesting repo branches. 
This repo is hosted on VSTS - devdiv.visualstudio.com

### CODE REVIEW

**Comment by Shyam-Gupta** [comment]
@dotnet-bot retest prtest/Ubuntu16.04/Debug please

File: src/AzureFunctionPackage/Functions/VstsCreateMergePRs/VstsCreateMergePRs.csproj [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.2.1" />
+    <PackageReference Include="System.IO.Compression.ZipFile" Version="4.3.0" />
+    <PackageReference Include="System.ValueTuple" Version="4.3.1" />
+    <PackageReference Include="WindowsAzure.ServiceBus" Version="3.3.2" />
+    <PackageReference Include="Microsoft.Azure.KeyVault" Version="2.0.6" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <AzureFunctionFiles Include="$(MSBuildThisFileDirectory)..\Common\auth.csx" />
+    <AzureFunctionFiles Include="$(MSBuildThisFileDirectory)..\Common\config.xml" />
```
Comment (brettfo) [inline src/AzureFunctionPackage/Functions/VstsCreateMergePRs/VstsCreateMergePRs.csproj]: This should be changed to `VstsPrConfig.xml`.

**Comment by Shyam-Gupta** [comment]
@dotnet-bot retest prtest/Ubuntu16.04/Debug please

**Comment by Shyam-Gupta** [comment]
Brett confirmed that the build failures are flaky and the built AzureFunction zip file has the new Azure function. Hence we are good to merge the PR.

**Pull request merged**